cluster/ClusterDatabaseManager.java

```diff
@@ -67,8 +67,8 @@ public void create(String name) {
         StringBuilder errors = new StringBuilder();
         for (String address : databaseManagers.keySet()) {
             try {
-                DatabaseProto.Database.Get.Res res = client.coreClient(address).call(
-                        () -> client.graknClusterRPC(address).databaseGet(DatabaseProto.Database.Get.Req.newBuilder().setName(name).build())
+                ClusterDatabaseProto.ClusterDatabaseManager.Get.Res res = client.coreClient(address).call(
```

Comment:
Oh, I never noticed this before, but shouldn't the construction of the `ClusterDatabaseProto.ClusterDatabaseManager.Get.Req` be moved to `RequestBuilder`?